scripts/install-contracts/index.js

```diff
@@ -0,0 +1,557 @@
+const fs = require('fs');
+const {
+  CasperServiceByJsonRPC,
+  Keys,
+  Contracts,
+  CasperClient,
+  CLValueBuilder,
+  RuntimeArgs,
+} = require('casper-js-sdk');
+const path = require('path');
+const {
+  installContract,
+  prepareContractCallDeploy,
+  waitForDeploy,
+  parseExecutionResult,
+  parseWriteContract,
+  stringToCLKey,
+} = require('./utils');
+
+const rawConfig = fs.readFileSync(path.resolve(__dirname, './config.json'), 'utf-8');
+
+const config = JSON.parse(rawConfig);
+
+const rpcAPI = new CasperServiceByJsonRPC(config.node_rpc_uri);
+const contractClient = new Contracts.Contract(new CasperClient(config.node_rpc_uri));
+
+async function main() {
+  const baseWasmPath = path.resolve(__dirname, '../../target/wasm32-unknown-unknown/release');
+
+  const status = await rpcAPI.getStatus();
+
+  const pk = Keys.Ed25519.loadKeyPairFromPrivateFile(config.private_key_path);
+
+  console.log(`
+    Info: Installing contracts:
+      1) VariableRepositoryContract
+      2) ReputationContract
+      3) DaoIdsContract
+      4) VaNftContract
+      5) KycNftContract
+  `);
+
+  let contractsConfig = [
+    {
+      name: 'VariableRepositoryContract',
+      wasmPath: `${baseWasmPath}/variable_repository_contract.wasm`,
+      args: [],
+      paymentAmount: '108880890230',
```

Comment:
I would move `paymentAmount` and `wasmPath` to the configuration because:
1. They may change during the development of the contracts (especially the payment amount)
2. As a side-effect it will make the config more readable (atm reading it raises a question "why do we have only some of the contracts here?")
3. It will be possible to add new contracts just updating the config file (atm it may require changes in the config and in the code)